src/components/partners/PartnersContent.tsx

```diff
@@ -0,0 +1,92 @@
+import { Box, Container, Grid } from '@mui/material'
+import Image from 'next/image'
+import { Fragment } from 'react'
+import { useTranslation } from 'react-i18next'
+import useMobile from '../../common/hooks/useMobile'
+import { organizationLogos, partners, techAndMarketingPartners } from './helpers/partnersData'
+import { SectionTitle, StyledGrid } from './Partners.styled'
+import PartnersSlider from './PartnersSlider'
+
+const PartnersContent = () => {
+  const LOGO_WIDTH = 181
+  const LOGO_HEIGHT = 99
+
+  const { t } = useTranslation()
+  const { mobile } = useMobile()
+
+  return (
+    <>
+      <Container maxWidth="lg">
+        <Grid container alignItems={'center'}>
+          {partners.map((partner) => (
+            <StyledGrid item xs={12} md={4} paddingTop={8} key={partner.name}>
+              <SectionTitle>{t(partner.title)}</SectionTitle>
+              <Image
+                alt={`${partner.name}-logo`}
+                src={`/img/partners/${partner.name}.svg`}
+                width={LOGO_WIDTH}
+                height={LOGO_HEIGHT}
+                priority
```

Comment:
the priority tag here causes a lot of warnings, could you check pls:
<img width="805" alt="Screenshot 2023-02-14 at 09 52 24" src="https://user-images.githubusercontent.com/7055304/218675798-0e8098d2-ca1d-41f8-bef3-6d2b9fe01fb1.png">

Reply:
I've added lazy loading to carousel component, but not sure if it'll work, I didn't have such warnings. Is it possible to check it again locally to be sure that the warning is gone?

Reply:
The lazy loading for the slider definitely helped, but there are still  some warnings in both Chrome/Firefox for these partners here. Removing the priority tag removes the warning. 

Also I think the problem with priority is that Nextjs Image is not setting the 'as' property correctly. So we can leave the priority like this and it will be fixed with the Nextjs upgrade later on.